### PR TITLE
Update quick started and self managed to reflect agreed content for quickstart on Hack Day session

### PR DESCRIPTION
**Pull request by dcs3spp**
Update quick started and self managed to reflect agreed content for quickstart on Hack Day session

[preview](https://deploy-preview-2843--tyk-docs.netlify.app/)

[quick start](https://deploy-preview-2843--tyk-docs.netlify.app/docs/nightly/getting-started/quick-start/)
[self managed installation](https://deploy-preview-2843--tyk-docs.netlify.app/docs/nightly/tyk-self-managed/install/)

### CODE REVIEW

File: tyk-docs/content/getting-started/quick-start.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ menu:
 weight: 1
 ---
 
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+
+Run these commands:
+
+1. `git clone https://github.com/TykTechnologies/tyk-pro-docker-demo && cd tyk-pro-docker-demo`
```
Comment (joshblakeley) [inline tyk-docs/content/getting-started/quick-start.md]: Can we change to something like 
\```bash
git clone https://github.com/TykTechnologies/tyk-pro-docker-demo && cd tyk-pro-docker-demo
\```

As that should get us a copy to clipboard option

It then looks like this:

```bash
git clone https://github.com/TykTechnologies/tyk-pro-docker-demo && cd tyk-pro-docker-demo
```

Comment (joshblakeley) [reply tyk-docs/content/getting-started/quick-start.md]: Applies to all commands

**Review by joshblakeley — CHANGES_REQUESTED** [review]
Small note but looks good!

File: tyk-docs/content/getting-started/quick-start.md [hunk 2]
```diff
@@ -8,6 +8,24 @@ menu:
...
+
+```bash
+git clone https://github.com/TykTechnologies/tyk-pro-docker-demo && cd tyk-pro-docker-demo
+```
+
+```bash
+docker-compose up
+```
+
+Then navigate to [http://localhost:3000](http://localhost:3000) and input the licence key we've emailed you
```
Comment (joshblakeley) [inline tyk-docs/content/getting-started/quick-start.md]: Worth pointing to https://tyk.io/sign-up/ if they dont yet have their key?

Comment (dcs3spp) [reply tyk-docs/content/getting-started/quick-start.md]: @joshblakeley Something like this?

```suggestion
Then navigate to [http://localhost:3000](http://localhost:3000) and input the licence key we've emailed you. If you do not have a key then please visit [https://tyk.io/sign-up/](https://tyk.io/sign-up/)
```

Comment (joshblakeley) [reply tyk-docs/content/getting-started/quick-start.md]: yep so that in both spots should be good

**Review by joshblakeley — APPROVED** [review]

File: tyk-docs/content/getting-started/quick-start.md [hunk 3]
```diff
@@ -8,6 +8,24 @@ menu:
...
+
+```bash
+git clone https://github.com/TykTechnologies/tyk-pro-docker-demo && cd tyk-pro-docker-demo
+```
+
+```bash
+docker-compose up
+```
+
+Then navigate to [http://localhost:3000](http://localhost:3000) and input the licence key we've emailed you. If you do not have a key then please visit [https://tyk.io/sign-up/](https://tyk.io/sign-up/)
```
Comment (letzya) [inline tyk-docs/content/getting-started/quick-start.md:25]: We don't tend to put links as-is in the docs unless it serves the purpose. Any reason for that? (usually, we use a button or a label. Check this https://github.com/search?q=repo%3ATykTechnologies%2Ftyk-docs+%22ttps%3A%2F%2Ftyk.io%2Fsign-up%22&type=code)

Comment (letzya) [reply tyk-docs/content/getting-started/quick-start.md:25]: Also - why not use the https://tyk.io/sign-up/#self directly?

**Pull request merged**